test/functional/collations.test.js

```diff
@@ -26,7 +27,7 @@ describe('Collation', function () {
       let commandResult;
       testContext.server.setMessageHandler(request => {
         const doc = request.document;
-        if (doc.ismaster || doc.hello) {
+        if (doc[LEGACY_HELLO_COMMAND] || doc.hello) {
```

Comment:
We do this code repeatedly, so perhaps make this a utility function that checks a document for `doc[LEGACY_HELLO_COMMAND] || doc.hello` in `src/utils.ts` that we can call from anywhere?